daemon/algod/api/algod.oas2.json

```diff
@@ -1693,18 +1693,48 @@
           "type": "string"
         },
         "sig-type": {
-          "description": "Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig",
+          "description": "Indicates what type of signature is used by this account, must be one of:\n* sig\n* msig\n* lsig\n* or null if unknown",
           "type": "string",
           "enum": [
             "sig",
             "msig",
             "lsig"
           ]
         },
+        "total-apps-opted-in": {
+          "description": "The count of all application local data (AppLocalState objects) stored in this account.",
+          "type": "integer"
+        },
+        "total-assets-opted-in": {
+          "description": "The count of all assets (AssetHolding objects) held by this account.",
+          "type": "integer"
+        },
+        "total-created-apps": {
+          "description": "The count of all apps (AppParams objects) created by this account.",
+          "type": "integer"
+        },
+        "total-created-assets": {
+          "description": "The count of all assets (AssetParams objects) created by this account.",
+          "type": "integer"
+        },
         "auth-addr": {
           "description": "\\[spend\\] the address against which signing should be checked. If empty, the address of the current account is used. This field can be updated in any transaction by setting the RekeyTo field.",
           "type": "string",
           "x-algorand-format": "Address"
+        },
+        "deleted": {
+          "description": "Whether or not this account is currently closed.",
+          "type": "boolean"
+        },
+        "created-at-round": {
+          "description": "Round during which this account first appeared in a transaction.",
+          "type": "integer",
+          "x-algorand-format": "uint64"
+        },
+        "closed-at-round": {
```

Comment:
1. the struct is shared between indexer and algod 
2. It seems to me that this is inconsistently implemented across the sdks. e.g. java and go sdks have this, python does not. 
So, changing this here will not change the sdks, since they have already implemented it (go and java) based on the indexer specs (and python does not have it for algod and indexer).

Reply:
I think it is a good rubric that only fields that are actually provided by the API should be listed in the API spec.